cpp/modmesh/mesh/StaticMesh_interior.cpp

```diff
@@ -46,6 +46,21 @@ struct FaceBuilder
     using uint_type = typename number_base::uint_type;
     using real_type = typename number_base::real_type;
 
+    size_t add_point(int_type icl, int_type ifc)
```

Comment:
I do not understand why do we need this type.  It is not useful in a mesh.  Could you please elaborate more?

Reply:
In gmsh parser I using the number of elements to pre-allocate buffer, this number is provide by msh file, but this number contains point type cell which is we want to ignore. 

In `FaceBuilder` constructor will use input cell type buffer to pre-allocate buffer and give a initial value with -1, then `FaceBuilder::populate()` will going to fill face information, but if I ignore point type cell, then the `fcnds` still has initial value -1, that will cause the index overflow in `FaceBuilder::make_ndfcs()`.

That’s why I need to add point type cell handler in `FaceBuilder`.

Reply:
Thanks.  It's clear now.  When I dealt with the older version of gmsh, I didn't remember it has node type.  Perhaps it's a new data format.